src/org/usfirst/frc/team2706/robot/commands/IntakeAndHold.java

```diff
@@ -0,0 +1,65 @@
+package org.usfirst.frc.team2706.robot.commands;
+
+import org.usfirst.frc.team2706.robot.JoystickMap;
+import org.usfirst.frc.team2706.robot.Robot;
+import org.usfirst.frc.team2706.robot.subsystems.Intake;
+
+import edu.wpi.first.wpilibj.AnalogInput;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class IntakeAndHold extends Command {
+
+    private Intake inhale;
+    private AnalogInput IR_sensor;
+    private double m_motorPower;
+    /**
+     * Allows us to use the methods in 'Intake'
+     */
+    public IntakeAndHold(double motorPower) {
+        inhale = Robot.intake;
+        this.requires(Robot.intake);
+        m_motorPower = motorPower;
+       
+    }
+    
+    /**
+     * I don't believe initialization is required 
+     */
+    public void initialize() {}
+    
+    /**
+     * Turns the motors on to suck in the cube
+     */
+    public void execute() {
+        System.out.println(inhale.readIRSensor());
```

Comment:
Use Log.d() instead, its our logging system

Reply:
Ok, I fixed it. I didn't know how the logging system worked...